server/credential_provider.go

```diff
@@ -4,8 +4,8 @@ import "sync"
 
 // interface for user credential provider
 // hint: can be extended for more functionality
-// =================================IMPORTANT NOTE===============================
-// if the password in a third-party credential provider could be updated at runtime, we have to invalidate the caching
+//
+// Note: if the password in a third-party credential provider could be updated at runtime, we have to invalidate the caching
```

Comment:
I would like you to suggest using this

```suggestion
//
// # Important Note
//
// if the password in a third-party credential provider could be updated at runtime, we have to invalidate the caching
```

Source https://tip.golang.org/doc/comment#headings